INSTALL.md

```diff
@@ -42,7 +42,7 @@ animated GIFs.
 
 **NOTE:**
 There are several options for using [GMT on non-UNIX systems](https://docs.generic-mapping-tools.org/latest/cookbook/non-unix-platforms.html)
-such as Windows, including [Windows Subsytem for Linux](https://docs.microsoft.com/en-us/windows/wsl/),
+such as Windows, including [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/),
```

Comment:
```suggestion
such as Windows, including [Windows Subsytem for Linux](https://docs.microsoft.com/en-us/windows/wsl/),
```
This will be fixed by https://github.com/GenericMappingTools/gmt/pull/5269, so I prefer to leave the change in that PR so that there's no merge conflict.